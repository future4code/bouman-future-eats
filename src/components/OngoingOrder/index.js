import React, { Component } from 'react';
import styled from 'styled-components'
import ClockImg from '../../img/clock.png'

import {connect} from 'react-redux'

const MainContainer = styled.div`
height: 118px;
background-color: #5cb646;
width: 100%;
display: flex;
img {
  width: 32px;
  height: 32px;
  margin: 24px;
}
justify-content: left;
align-items: center;
`

const TextDiv = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
`

const StyledP = styled.p `
  width: 100%;
  height: 18px;
  font-family: Roboto;
  font-size: 16px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
  letter-spacing: -0.39px;
  color: #ffffff;
  margin: 0;
  margin-bottom: 8px;

`



function OnGoingOrder (props){

  

  return(
    <MainContainer>
      <img src={ClockImg}/>
      <TextDiv>
        <StyledP>Pedido em andamento</StyledP>
        <StyledP style={{color: "black"}}> {props.restaurant}</StyledP>
        <StyledP style={{color: "black" , fontWeight: "bold"}} >SUBTOTAL R${props.totalPrice}</StyledP>
      </TextDiv>
    </MainContainer>
  )
}

export default OnGoingOrder