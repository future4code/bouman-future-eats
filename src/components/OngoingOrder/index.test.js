import React from 'react'
import renderer from 'react-test-renderer'
import OngoingOrder from './index'

test("Ongoing order snapshot", ( ) => {
    const tree= renderer.create(<OngoingOrder text={''}/>).toJSON()

    expect(tree).toMatchSnapshot()
})