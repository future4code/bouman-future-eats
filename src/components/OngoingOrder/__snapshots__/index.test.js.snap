// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ongoing order snapshot 1`] = `
<div
  className="sc-bdVaJa jCOqdb"
>
  <img
    src="clock.png"
  />
  <div
    className="sc-bwzfXH brfdCc"
  >
    <p
      className="sc-htpNat iInFLm"
    >
      Pedido em andamento
    </p>
    <p
      className="sc-htpNat iInFLm"
      style={
        Object {
          "color": "black",
        }
      }
    >
       Bullguer Vila Madalena
    </p>
    <p
      className="sc-htpNat iInFLm"
      style={
        Object {
          "color": "black",
          "fontWeight": "bold",
        }
      }
    >
      SUBTOTAL R$67.00
    </p>
  </div>
</div>
`;
